content/billing/pricing.md

```diff
@@ -5,12 +5,12 @@ weight: 2
 
 ## Free plan
 
-The free plan allows you to use 500 minutes per month on macOS standard machines on a personal account. Your 500 free minutes will be reset on the 1st of each month. Free minutes are not available if you are using a Team.
+The free plan allows you to use 500 minutes per month on macOS M1 standard machines on a personal account. Your 500 free minutes will be reset on the 1st of each month. Free minutes are not available if you are using a Team.
 
 To start using Codemagic for free, [sign up here](https://codemagic.io/signup). 
 ## Pay as you go pricing
 
-Pay as you go pricing allows you to pay for only what you use. 
+Pay as you go pricing allows you to pay for only what you use. You will never be charged more than $299/month on Codemagic's pay as you go plan.
```

Comment:
Should this be explained in more detail so that it's clear they can continue using the service after they use it for 299?

Reply:
updated to say that you can continue to build.